services/worker-manager/schemas/v1/worker-full.yml

```diff
@@ -53,6 +53,12 @@ properties:
     type: string
     enum: ["requested", "running", "stopped"]
 
+  capacity:
+    title: Worker Capacity
+    description: Number of tasks this worker can handle at once
```

Comment:
Since this isn't required in `createWorker`, it should probably specify a default value in the docs.

Reply:
Ah shoot, I'll make it required actually.

Reply:
Making it required is a breaking change to the REST API, though.  That's fine, just upgrade the changelog to major :)